src/config.rs

```diff
@@ -232,7 +232,7 @@ macro_rules! providers {
 }
 
 providers! {
-    Discord: "https://discordapp.com/api/oauth2/authorize", "https://discordapp.com/api/oauth2/token",
+    Discord: "https://discord.com/api/oauth2/authorize", "https://discord.com/api/oauth2/token",
```

Comment:
Per https://discord.com/developers/docs/topics/oauth2#shared-resources-oauth2-urls, seems this might preferably be:

`https://discord.com/oauth2/authorize`

Reply:
My first reaction was that this had to be a typo in the docs, but I tested it and it works.

Reply:
A bit strangely, the two endpoints respond to an empty request with different error messages - one complains about missing `client_id`, the other about missing `scope`. But I would *expect* both to behave the same way for valid inputs.